Resources/Prototypes/_LostParadise/Maps/Plot_113.yml

```
@@ -0,0 +1,74 @@
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'DV'
+        - type: GridSpawn
+          groups:
+            mining:
+              paths:
+              - /Maps/_LostParadise/Shuttles/Salvage_113.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+          #service
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)